### PR TITLE
Tree Asset Decomp for object_tree

### PR DESCRIPTION
**Pull request by SeDan01**
Renamed DLists, Textures, and col.

### CODE REVIEW

File: assets/xml/objects/object_tree.xml [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
     <File Name="object_tree" Segment="6">
-        <DList Name="object_tree_DL_000680" Offset="0x680" />
-        <DList Name="object_tree_DL_0007C8" Offset="0x7C8" />
-        <Texture Name="object_tree_Tex_0008B8" OutName="tex_0008B8" Format="rgba16" Width="32" Height="32" Offset="0x8B8" />
-        <Texture Name="object_tree_Tex_0010B8" OutName="tex_0010B8" Format="rgba16" Width="32" Height="32" Offset="0x10B8" />
-        <Collision Name="object_tree_Colheader_001B2C" Offset="0x1B2C" />
+        <DList Name="gTreeBodyDL" Offset="0x680" />
+        <DList Name="gTreeLeavesDL" Offset="0x7C8" />
+        <Texture Name="gTreeBodyTex" OutName="tex_0008B8" Format="rgba16" Width="32" Height="32" Offset="0x8B8" />
+        <Texture Name="gTreeLeavesTex" OutName="tex_0010B8" Format="rgba16" Width="32" Height="32" Offset="0x10B8" />
```
Comment (hensldm) [inline assets/xml/objects/object_tree.xml:6]:
```suggestion
        <Texture Name="gTreeBodyTex" OutName="tree_body" Format="rgba16" Width="32" Height="32" Offset="0x8B8" />
        <Texture Name="gTreeLeavesTex" OutName="tree_leaves" Format="rgba16" Width="32" Height="32" Offset="0x10B8" />
```
Only thing left is filling out the `OutName`. They just use the same name as the texture, but drop the `g` and `tex`, as well as use snakecase. These are the filenames that ZAPD extracts the pngs to.

**Review by hensldm — APPROVED** [review]

**Review by jpburnett — APPROVED** [review]

**Comment by jpburnett** [comment]
🌳

**Review by AngheloAlf — APPROVED** [review]
Well done!

**Review by tom-overton — APPROVED** [review]

**Comment by engineer124** [comment]
Great work! PR looks good.

I'd like to mention that we are trying to get everybody who has contributed and/or wants to contribute to MM decomp to agree to licensing the repo before merging PRs from people who has not yet agreed to it. It is better explained here: https://github.com/louist103/mm-licensing/issues/1.

Could you please read about it and post in that issue if you agree or not. Thanks!

**Review by engineer124 — APPROVED** [review]

**Pull request merged**